charts/pxc-db/templates/cluster.yaml

```diff
@@ -38,9 +38,13 @@ spec:
   {{- else }}
   logCollectorSecretName: {{ include "pxc-database.fullname" . }}-log-collector
   {{- end }}
+  {{- if .Values.initImage }}
+  initImage: {{ .Values.initImage }}
```

Comment:
is operator able to handle empty `initImage` field without crash or error?

Reply:
Yes, by default it uses same as operator image, this is specified only if some custom initImage needed.

Reply:
I have simplified it now and removed the `if`.